inc/Engine/Preload/Controller/ClearCache.php

```diff
@@ -30,13 +30,19 @@ public function __construct( Cache $query ) {
 	 */
 	public function partial_clean( array $urls ) {
 		foreach ( $urls as $url ) {
-			if ( ! $this->is_excluded( $url ) ) {
+			if ( $this->is_excluded( $url ) ) {
+				continue;
+			}
```

Comment:
No need to use this check as we already hooking to the filter here: 
https://github.com/wp-media/wp-rocket/pull/5433/files#diff-13bb6ae74d8bc004dd6fbdb9a3d705ceb0c263bb9ffa6e4df10ebc19190589afR116-R119
and add the callback `add_cache_reject_uri` which gets the option values and add to the filter return.